openmrs-watcher/src/main/java/org/openmrs/eip/mysql/watcher/route/WatcherShutdownRoute.java

```diff
@@ -0,0 +1,24 @@
+package org.openmrs.eip.mysql.watcher.route;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.openmrs.eip.Utils;
+import org.openmrs.eip.mysql.watcher.CustomFileOffsetBackingStore;
+import org.springframework.stereotype.Component;
+
+/**
+ * Route that is invoked when the watcher application is shutdown. - Disables the
+ * CustomFileOffsetBackingStore - Sends a notification to the configured recipients
+ */
+@Component
+public class WatcherShutdownRoute extends RouteBuilder {
```

Comment:
Is this change relevant to the ticket scope?

Reply:
No alterations have been made, only rewritten using Java DSL. I encountered an issue while attempting to register/call `CustomFileOffsetBackingStore`.

Reply:
Could you briefly document this issue for the record, maybe in the description?